### PR TITLE
[Crystal] Add runcobo

### LINKED ISSUE #2958: [Crystal] Add runcobo
Hi @shootingfly,

Would you mind to add (or consent that we add) [runcobo](https://github.com/runcobo/runcobo) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

### PR DESCRIPTION
**Pull request by waghanza**
Hi @shootingfly,

The route `"/user/:id"` seems not to work. I have `Missing param name: "id" (KeyError)`.

Regards,

------------
Closes #2958

### CODE REVIEW

File: crystal/runcobo/src/server.cr [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class CreateUser < BaseAction
+  post "/user"
+
+  call do
+    render_plain ""
+  end
+end
+class ShowUser < BaseAction
+  get "/user/:id"
+  query NamedTuple(id: Int32)
```
Comment (shootingfly) [inline crystal/runcobo/src/server.cr:19]: url NamedTuple(id: Int32)

Comment (waghanza) [reply crystal/runcobo/src/server.cr:19]: ❤️

**Comment by waghanza** [comment]
@shootingfly May I ask for your code review ? can I put you in `CODEOWNERS`, so as you can be noticed for each change of `runcobo` here ?

**Comment by shootingfly** [comment]
Okay. No problem.

**Pull request merged**